tuning/sensorless.md

```diff
@@ -0,0 +1,477 @@
+2. Physically disconnect the X and Y endstop connections from the MCU.
+3. Plug in jumpers on the `DIAG` ports as shown below (**J16** and **J17**). We use these ports since thats the corresponding diag jumper for where our XY steppers are connected.
+   
+    ![image](images/sensorless/OctoDiag.png)
+   
+5. Start up the printer and wait for it to boot
+6. MAKE A BACKUP OF YOUR CONFIG (or better yet go do [this](https://docs.vorondesign.com/community/howto/EricZimmerman/BackupConfigToGithub.html))
+
+## Choose your own adventure
+
+Here we come to a choice. Do you want to use stock Klipper and macros to control homing, or do you want to switch to an alternate version of Klipper, named Danger Klipper (do not worry. its not really dangerous)?
+
+The Danger Klipper route will make things simpler from the configuration side, but in the end, your printer will home either way without too much fuss. Pick ONE of the two options below, either Stock Klipper or Danger Klipper.
+
+### Stock Klipper
+
+In this path, you will need to download some macros dependong on your printer. If you do not have a v0 or a 2.4, you can tweak whichever makes the most sense for you. When in doubt, ask in the Voron Discord channel for your printer
+
+1. Create a new file in the location of your choosing named `homing.cfg`
+2. Open another browser tab and open [this](https://github.com/EricZimmerman/Voron24/blob/PreKlippain/macros/helpers/homing.cfg) page if you have a 2.4, or [this](https://github.com/EricZimmerman/Voron02/blob/PreKlippain/macros/helpers/homing.cfg) page if you have a v0.
+3. Edit `homing.cfg` 
+4. **Add** the macro from step 7 to `homing.cfg`. **_Be sure to edit the coordinates as needed for your bed in the Z section (the line that reads **G1 X175 Y175 F15000**). IE to the center of the bed, your endstop pin for z, etc._**
+5. Save `homing.cfg` 
+6. Edit your `printer.cfg` file
+7. If you have a `[safe_z_home]` section, find it and comment it out as we will be using homing override as found in `homing.cfg`.
+
+The `[homing_override]` block we have in the above macros is now going to be responsible for all homing, whether via the buttons in Mainsail or Fluidd, or via commands like G28 X, etc. This override allows us, the end user, to customize how the homing operation happens. When using sensorless, this is important, as it lets us adjust things like the current used for homing, etc. More on this later.
+
+### Danger Klipper
+
+While it sounds dangerous, there is nothing to be afraid of! Choosing this route means you do not need to do any of the macro stuff as listed above, as we can just simply add a new property in our X and Y driver section to handle homing current. Thats it!
+
+If you want to switch to Danger Klipper, now is the time to do so. You can follow [this](https://github.com/EricZimmerman/VoronTools/blob/main/DangerKlipper.md) to make the switch! Switching will not make you change anything else about your configutation.
```

Comment:
- links should have a bigger tap target for users on mobile than "this"
- configutation -> configuration
```suggestion
If you want to switch to Danger Klipper, now is the time to do so. You can [follow this other guide](https://github.com/EricZimmerman/VoronTools/blob/main/DangerKlipper.md) to make the switch! Switching will not make you change anything else about your configuration.
```